the-telephone-app-fe/src/assets/index.css

```diff
@@ -0,0 +1,223 @@
+@import url('https://fonts.googleapis.com/css2?family=Fredoka:wght@400;500;600;700&display=swap');
+:root {
+    background-color: #FFF9EB;
+}
+body {
+}
+
+.row {
+    --bs-gutter-x: 0;
+}
+.h-2 {
+    height: 2%;
+}
+.h-10 {
+    height: 10%;
+}
+.h-13 {
+    height: 13%;
+}
+.h-15 {
+    height: 15%;
+}
+.h-20 {
+    height: 20%;
+}
+.h-80 {
+    height: 80%;
+}
+.h-70 {
+    height: 70%;
+}
+.h-72 {
+    height: 70%;
+}
+.h-65 {
+    height: 65%;
+}
+.h-87 {
+    height: 87%;
+}
+.h-67 {
+    height: 67%;
+}
+.y {
+    display: flex;
+    align-items: center;
+}
+.center {
+    display: flex;
+    justify-content: center;
+}
+.right {
+    display: flex;
+    justify-content: right;
+}
+.left {
+    display: flex;
+    justify-content: left;
+}
+.all {
+    width: 100%;
+    height: 100%;
+}
+.main {
+    width: 80%;
+    height: 80%;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+}
+.img-logo {
+    height: 116px;
+}
+.button {
+    padding: 10px 20px;
+    border-radius: 8px;
+    background-color: #C3C9E8;
```

Comment:
Define variable for colors that were used many times.
Create a css file for this then import it in the css files that use these colors.
Huy created some variable in his pull request. When it merged you can move them to a color file that will be import to use in your and his files instead.